### PR TITLE
Add xml And xPath prebuilt function

### PR DESCRIPTION
**Pull request by cosmicshuai**
Add xml and xPath pre-built functions to Adaptive-Expressions

xml and xPath can run in both browser and nodejs environment.

### CODE REVIEW

**Comment by cosmicshuai** [comment]
Removed type definition of x2js package due to it use 'Omit' in index.d.ts. This only work for typescript version >= 3.5.0
This will cause yarn test:consumer failure. 
```
"../../node_modules/@types/xml2js/index.d.ts(95,23): error TS2304: Cannot find name 'Omit'.\n",
```
```

File: libraries/adaptive-expressions/src/builtinFunctions/xml.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+export class XML extends ExpressionEvaluator {
+    /**
+     * Initializes a new instance of the [EOL](xref:adaptive-expressions.EOL) class.
+     */
+    public constructor() {
+        super(ExpressionType.XML, XML.evaluator(), ReturnType.String, FunctionUtils.validateUnary);
+    }
+
+    /**
+     * @private
```
Comment (joshgummersall) [inline libraries/adaptive-expressions/src/builtinFunctions/xml.ts:27]: Our JSDoc lint rule will only require comments on public or protected methods, not private ones. This comment block should be omitted.

**Review by joshgummersall — CHANGES_REQUESTED** [review]
Just a quick ask to remove unnecessary private method JSDoc comments.

File: libraries/adaptive-expressions/src/builtinFunctions/xml.ts (hunk 1 above)
Comment (cosmicshuai) [reply libraries/adaptive-expressions/src/builtinFunctions/xml.ts]: removed

**Review by joshgummersall — APPROVED** [review]

**Pull request merged**